app-frontend/src/app/pages/settings/settings.html

```diff
@@ -0,0 +1,15 @@
+<div class="container column-stretch dashboard">
+  <div class="sidebar">
+    <div class="sidebar-scrollable">
+      <nav>
+        <a ui-sref="settings.profile" ui-sref-active="active">Profile</a>
+        <a ui-sref="settings.account" ui-sref-active="active">Account</a>
+        <a ui-sref="settings.tokens" ui-sref-active="active">API tokens</a>
```

Comment:
No, this is fine. I just wasn't sure if there maybe were prototypes for them and they were expected to work. :+1: